packages/ra-core/src/controller/list/useListParams.ts

```diff
@@ -77,17 +77,18 @@ export interface ListParams {
  * } = listParamsActions;
  */
 export const useListParams = ({
-    resource,
-    filterDefaultValues,
-    sort = defaultSort,
-    perPage = 10,
+    customStoreKey = '',
     debounce = 500,
     disableSyncWithLocation = false,
+    filterDefaultValues,
+    perPage = 10,
+    resource,
+    sort = defaultSort,
 }: ListParamsOptions): [Parameters, Modifiers] => {
     const location = useLocation();
     const navigate = useNavigate();
     const [localParams, setLocalParams] = useState(defaultParams);
-    const storeKey = `${resource}.listParams`;
+    const storeKey = `${resource}${customStoreKey}.listParams`;
```

Comment:
No, storeKey should be the provided storeKey, or, if not provided, `${resource}.listParams`